packages/teams-js/src/public/app.ts

```diff
@@ -814,6 +814,10 @@ export namespace app {
 
   /**
    * Notifies the frame that app has loaded and to hide the loading indicator if one is shown.
+   * 
+   * @remarks
```

Comment:
>@remarks [](http://example.com/codeflow?start=5&length=8)

Should these remarks be specifically on the `notifyAppLoaded` method like this? It seems like the documentation is generally about how loading indicator is handled and not specific to this one function.